client/homebrew/pages/editPage/editPage.jsx

```diff
@@ -247,16 +247,17 @@ const EditPage = createClass({
 			});
 		if(!res) return;
 
-		this.savedBrew = res.body;
+		this.savedBrew = {
```

Comment:
The old version uses the whole brew (or stub) returned from the server. This only uses the googleId, editId, shareId, and version. Are these the only values that can change as a result of the `/update` API call? Just want to double-check we don't risk losing some brew metadata when saving here.

Reply:
Clarified over Gitter: old code was resulting in `this.savedBrew` and `this.state.brew` not being updated with the same contents after a save. This PR guarantees both are identical after a save.